src/io/files.js

```diff
@@ -714,6 +714,88 @@ p5.prototype.httpGet = function () {
  * @param  {function}      [errorCallback] function to be executed if
  *                                    there is an error, response is passed
  *                                    in as first argument
+ *
+ * @example
+ * <div><code>
+ * // Examples use jsonplaceholder.typicode.com for a Mock Data API
+ *
+ * var url = 'https://jsonplaceholder.typicode.com/posts';
```

Comment:
I assume this service doesn't have rate limiting?

Reply:
I don't think so. Can't find any documentation and my own testing seems to agree